### PR TITLE
Added TextArea component

### PR DESCRIPTION
**Pull request by nicokoelewijn**
### This PR:

Contains the all new Textarea component.

proposed release: `minor`

**Checklist** 🛡
- [x] I have exported my addition from `src/index.ts` (check if not applicable)
- [x] Appropriate tests have been added for my functionality (check if not applicable)

**Adds** ✨
- Component `Textarea`

### CODE REVIEW

File: src/components/Textarea/index.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React, { Component, ChangeEvent } from 'react';
+import StyledTextarea, { StyledTextareaWrapper } from './style';
+import InlineNotification from '../InlineNotification';
+import SeverityType from '../../types/SeverityType';
+import trbl from '../../utility/trbl';
+import Box from '../Box';
+
+type PropsType = {
+    rows?: number;
```
Comment (CarloPalinckx) [inline src/components/Textarea/index.tsx]: maybe `lines` instead of `rows` would make more sense here?

File: src/components/Textarea/index.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    value: string;
+    name: string;
+    id?: string;
+    resizeable?: boolean;
+    disabled?: boolean;
+    feedback?: {
+        severity: SeverityType;
+        message: string;
```
Comment (CarloPalinckx) [inline src/components/Textarea/index.tsx]: Isn't message going to be optional in some other proposal?

File: src/components/Textarea/index.tsx (hunk 1 above)
Comment (nicokoelewijn) [reply src/components/Textarea/index.tsx]: rows is the html default (https://www.w3schools.com/tags/tag_textarea.asp)

**Comment by CarloPalinckx** [comment]
Depends on #220 to be merged

File: src/themes/MosTheme/MosTheme.theme.ts [hunk 3]
```diff
@@ -758,6 +782,7 @@ const theme: ThemeType = {
             } 20px )`,
         },
     },
+
```
Comment (CarloPalinckx) [inline src/themes/MosTheme/MosTheme.theme.ts]: Nit: whitespace

**Pull request merged**